### PR TITLE
Add Tove to the list of editors

### PR DESCRIPTION
**Pull request by eladalon1983**
Adding @tovepet to the list of editors.

### CODE REVIEW

**Review by alvestrand — COMMENTED** [review]
I support this change (as long as Tove is willing to do it).

**Comment by tovepet** [comment]
Yes, I'm happy to take this on.

**Review by jan-ivar — APPROVED** [review]
That would be great! Thanks!

File: respec-config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ var respecConfig = {
   editors:  [
     { name: "Elad Alon", company: "Google", w3cid: 118124 },
     { name: "Jan-Ivar Bruaroey", company: "Mozilla", w3cid: 79152}
```
Comment (jan-ivar) [inline respec-config.js:8]:
```suggestion
    { name: "Jan-Ivar Bruaroey", company: "Mozilla", w3cid: 79152},
```

Comment (henbos) [inline respec-config.js:8]: ,

**Review by henbos — APPROVED** [review]

**Pull request merged**